policy-service/pom.xml

```diff
@@ -155,6 +155,10 @@
             <version>${alpakka.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.aspectj</groupId>
+            <artifactId>aspectjweaver</artifactId>
+        </dependency>
```

Comment:
Is this dependency needed for anything?

Reply:
Not anymore.  I was originally using AspectJ for a Aspect Oriented Programming approach to handling the errors.  Changed the approach to latch it onto the proxy.  I'll removed these.